### DISCUSSION TITLE
How to render mesh triangles double-sided?

### DISCUSSION THREAD

**Discussion by Hex4dec** [raysan5/raylib#4289] [Q&A] [answered]
I'm rendering a mesh and it seems to be randomly rendering the front or back face. I want the triangles in the mesh to be double-sided without specifying normals (yea, i don't have normals, i don't need fancy lighting) or specifiying the triangles twice in vertex data.

**Comment by raysan5** [comment, marked as answer]
@Hex4dec You can remove backface culling using `rlgl` function: `rlDisableBackfaceCulling()`.
